tests/test_files/html/block_quotes_action_link.html

```diff
@@ -0,0 +1,12 @@
+<blockquote>
+<p><a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ"><img alt="call to action img" src="https://dev-va-gov-assets.s3-us-gov-west-1.amazonaws.com/img/vanotify-action-link.png" class="action_link"><b>action link</b></a></p>
+</blockquote>
+<p><a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ"><img alt="call to action img" src="https://dev-va-gov-assets.s3-us-gov-west-1.amazonaws.com/img/vanotify-action-link.png" class="action_link"><b>action link</b></a></p>
```

Comment:
Is this missing a `<blockquote>`. I see we have a `</blockquote>`, but not an opening tag.

Reply:
there's a `</blockquote>` on line 5

Reply:
Yes!  Thank you.